### DISCUSSION TITLE
New User Save location

### DISCUSSION THREAD

**Discussion by GobSh1te** [localsend/localsend#1454] [Discussion (English, International)]
Hello I'm new to localsend and although it is working I find it terribly hard to use when sending files to my phone. Instead of files going to a specific folder it saves the file in different folders depending on the file type. Is it possible to save to a specific folder?

**Comment by Tienisto** [comment]
Usually, it saves into the Downloads folder. However, media files are saved into the folder where other media files (like photos) are located. You can turn this feature off by disabling "Save media to gallery".
